pages/settings/settings.js

```diff
@@ -500,3 +500,88 @@ proxyTypeInput.addEventListener('change', e => {
 })
 
 proxyInputs.forEach(item => item.addEventListener('change', e => setProxy(e.target.name, e.target.value)))
+
+settings.get('customBangs', (value) => {
+    const bangslist = document.getElementById('custom-bangs');
+    
+    if(value) { value.forEach(function(bang) {
+        bangslist.appendChild(createBang(bang.phrase, bang.snippet, bang.redirect))
+        bangslist.appendChild(document.createElement('br'));
+    })
+}
+})
+
+
+document.getElementById('add-custom-bang').onclick = function() {
```

Comment:
Use `addEventListener` - the reason is that something else could override `onclick` (not likely to happen in this case, but still good practice).

Reply:
Got it